Makefile

```diff
@@ -94,6 +101,10 @@ clean:
 	rm -rf $(OUT)
 	cd images && $(MAKE) clean
 
+.PHONY: dev-site-recreate
+dev-site-recreate:
+	-gravity site delete --force --insecure --debug
+	gravity site create --insecure --debug --app="$(PACKAGE)"
```

Comment:
app should be create with this action, otherwise you will got error that files in `/var/lib/gravity/local/packages/unpacked/gravitational.io/stolon-app/0.0.4/resources` are not found.
During site create with app gravity properly unpack tar.gz